metacat-main/src/main/java/com/netflix/metacat/main/configs/ServicesConfig.java

```diff
@@ -73,7 +74,9 @@ public UserMetadataService userMetadataService(
         final Config config,
         final MetacatJson metacatJson
     ) {
-        return new MysqlUserMetadataService(dataSourceManager, metacatJson, config);
+        return config.userMetadataServiceEnabled()
```

Comment:
Along with my previous comment this really limits us to ONLY having a mysql or default implementation. Which is not a very good or extensible design. I understand this is the case right now but we should be able to plug in different implementations without modifying this code.